src/Databases/MySql.php

```diff
@@ -60,6 +63,26 @@ public function dontUseSingleTransaction()
     }
 
     /**
+     * @return $this
+     */
+    public function useSkipComments()
```

Comment:
I think `skipComments` sound friendlier than `useSkipComments`. If you agree, please rename all usages.